counterpoll/main.py

```diff
@@ -382,6 +394,52 @@ def disable(ctx):
     fc_info['FLEX_COUNTER_STATUS'] = 'disable'
     ctx.obj.mod_entry("FLEX_COUNTER_TABLE", "FLOW_CNT_ROUTE", fc_info)
 
+# ENI counter commands
+@cli.group()
+@click.pass_context
+def eni(ctx):
+    """ ENI counter commands """
+    ctx.obj = ConfigDBConnector()
+    ctx.obj.connect()
+
+@eni.command()
+@click.argument('poll_interval', type=click.IntRange(1000, 30000))
+@click.pass_context
+def interval(ctx, poll_interval):
+    """ Set eni counter query interval """
+    if not is_dpu(ctx.obj):
```

Comment:
Can't this be verified during creation of the context in `eni` function?

Reply:
Agree. Lets have the check at eni context